cmd/skaffold/app/cmd/build_test.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"io"
+	"io/ioutil"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/flags"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestQuietFlag(t *testing.T) {
+	mockCreateRunner := func(c context.Context, buildOut io.Writer) ([]build.Artifact, error) {
+		return []build.Artifact{{
+			ImageName: "gcr.io/skaffold/example",
+			Tag:       "test",
+		}}, nil
+	}
+
+	orginalCreateRunner := createRunnerAndBuildFunc
```

Comment:
same as below (Sorry I read from bottom to top...)

Reply:
same. they are set to mock functions in the test runs.